src/igvfd/schemas/changelogs/sample.md

```diff
@@ -0,0 +1,5 @@
+## Changelog for biosample.json
+
+### Minor changes since version 1
+
+* Bumped date_obtained property from biosample to sample.
```

Comment:
please see my comment on the changelog comment for Biosample, apply similar logic here.